tests/publiccloud/upload_image.pm

```diff
@@ -31,14 +32,16 @@ sub run {
         return;
     }
 
-    # Download the given image. Check for 404 errors and make them better visible
-    my $cmd = "wget --no-check-certificate $img_url -O $img_name";
-    my $rc = script_run("$cmd 2>download.txt", timeout => 60 * 10);
+    # Download the given image via wget. Note that by default wget retries 20 times before giving up
+    my $cmd = "wget --no-check-certificate --retry-connrefused --retry-on-host-error $img_url -O $img_name";
+    # A generious timeout is required because downloading up to 30 GB (Azure images) can take more than an hour.
+    my $rc = script_run("$cmd 2>&1 | tee download.txt", timeout => 120 * 60);
     if ($rc != 0) {
-        upload_logs("download.txt");
```

Comment:
Is this useless? IMO better than dumping the output in `record_info`

Reply:
The output containts 99% only the download progress, so IMHO this is completely useless to upload.
I might need to adjust the `script_output` as well, though :thinking: Will continue on Monday.

Reply:
I added the `upload_logs` again and cropped the output in record_info to the last 10 lines of that log to avoid a HUGE wall of text.